src/components/dialog/dialog.ts

```diff
@@ -46,6 +46,12 @@ export class MDialog extends ModulVue implements PortalMixinImpl {
     @Prop({ default: true })
     public paddingFooter: boolean;
 
+    $refs: {
```

Comment:
Bonne idée de faire ça! Dans ce cas on pourrait enlever le casting qu'on faisait systématiquement. Ex ligne 86:
`return this.$refs.article as HTMLElement;`

Reply:
Ouais je voulais pas trop toucher a dialog mais je vais passer la grate :-)

Reply:
Le chasse neige a passé :-)